{{ cookiecutter.project_name }}/localdev/app.ipynb.example

```diff
@@ -0,0 +1,47 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import django\n",
+    "from django.contrib.auth import get_user_model\n",
+    "\n",
+    "ABS_PROJECT_ROOT_PATH = \"/src\"\n",
+    "BASE_DJANGO_APP_NAME = os.environ.get(\"BASE_DJANGO_APP_NAME\")\n",
+    "\n",
+    "if not ABS_PROJECT_ROOT_PATH in sys.path:\n",
+    "    sys.path.insert(0, ABS_PROJECT_ROOT_PATH)\n",
+    "os.environ.setdefault('DJANGO_SETTINGS_MODULE', '{}.settings'.format(BASE_DJANGO_APP_NAME))\n",
+    "django.setup()\n",
+    "\n",
+    "User = get_user_model()"
+   ]
```

Comment:
with the app now named 'main', the content should be modified to this (get rid of `BASE_DJANGO_APP_NAME`):
```python
import os
import sys
import django
from django.contrib.auth import get_user_model

ABS_PROJECT_ROOT_PATH = "/src"
if not ABS_PROJECT_ROOT_PATH in sys.path:
    sys.path.insert(0, ABS_PROJECT_ROOT_PATH)
os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'main.settings')
django.setup()

User = get_user_model()
```